torch/_dynamo/eval_frame.py

```diff
@@ -360,9 +359,84 @@ def first_real_inst_idx(code):
     raise RuntimeError("RESUME instruction not found in code")
 
 
+# Converts a CodePart structure, which carries our
+# re-compilation code, source, and more, into a
+# plan for how to mark certain dims as dynamic.
+def dynamic_plan_from_code_part(code_part, dynamic_plan):
+    from torch._dynamo.source import TensorProperty, TensorPropertySource
+
+    if (
+        code_part is None
+        or config.automatic_dynamic_shapes_strategy == config.DYNAMIC_SHAPE_STRATEGY.OFF
+    ):
+        return dynamic_plan
```

Comment:
There's no point in returning this, right? You MUST mutate the original dict

Reply:
Correct